paddle/fluid/framework/ir/memory_optimize_pass/reference_count_pass.cc

```diff
@@ -265,24 +223,27 @@ ExtractComputationOpFromLastLivedVar(details::VarHandle *var, size_t scope_idx,
   std::unordered_set<details::ComputationOpHandle *> computation_op;
```

Comment:
Maybe `computation_ops` is more appropriate, compared to `candidates`.

Reply:
Done.